cirq/json_resolver_cache.py

```diff
@@ -153,5 +153,8 @@ def two_qubit_matrix_gate(matrix):
         'sympy.Float': lambda approx: sympy.Float(approx),
         'sympy.Integer': sympy.Integer,
         'sympy.Rational': sympy.Rational,
+        'sympy.pi': lambda: sympy.pi,
```

Comment:
Could we add a test (or modify one of the existing tests) to use one of these symbols?

Thanks for contributing!

Reply:
NP, will add it right away!
Looping in @jacksonwb, this PR should fix the serialization issues with parametrized circuits

Reply:
@dstrain115 added some test, let me know what you think!

Reply:
This comment has been addressed.